run_tests.R

```diff
@@ -2,7 +2,7 @@
 # This script runs most tests in nimble/inst/tests/ prioritized by duration.
 
 # Avoid running these blacklisted tests, since they take too long.
-blacklist <- c('test-Math2.R', 'test-Mcmc2.R', 'test-Mcmc3.R', 'test-Filtering2.R', 'test-ADfunctions.R', 'test-ADmodels.R')
+blacklist <- c('test-Math2.R', 'test-Mcmc2.R', 'test-Mcmc3.R', 'test-Filtering2.R', 'test-ADfunctions.R', 'test-ADmodels.R','test-refactorSizeProcessing.R')
```

Comment:
Why is the test blacklisted? Does it currently fail? This knowledge deserves to live as a comment in the code:
```r
blacklist <- c(blacklist, 'test-refactorSizeProcessing.R')  # These tests of an experimental refactoring currently fail.
```